MVP/defaults-build.yaml

```diff
@@ -63,24 +49,9 @@
         - junit:
             results: 'J:*.xml'
         - ci-publisher:
-            message-type: 'Tier1TestingDone'
-            message-properties: |
-                CI_TYPE=ci-metricsdata
-            message-content: |
-                {{
-                  "create_time": "$create_time",
-                  "tests": $TESTS,
-                  "CI_tier": "1",
-                  "owner": "$job_owner",
-                  "build_type": "",
-                  "base_distro": "",
-                  "completion_time": "$(date --utc +%FT%TZ)",
-                  "component": "$name-$version-$release",
-                  "jenkins_job_url": "$JOB_URL",
-                  "jenkins_build_url": "$BUILD_URL",
-                  "brew_task_id": "$task_id",
-                  "job_names": "$JOB_NAME",
-                }}
+            message-type: null
+            message-properties: null
+            message-content: null
```

Comment:
Should we just remove the ci-publisher instead of setting these values to 'null', until a newer ci-publisher can take its place?

Reply:
Removed.